### ISSUE TITLE
trained model

### ISSUE THREAD

**Issue by rplawate** [kurapan/CRNN#1]
Can you please provide pre trained model ?

**Comment by kurapan** [comment]
Sorry for replying late.
I uploaded a model pretrained on Synth90k.
I put the link into the README.

**Comment by rplawate** [comment]
Thanks alot! 
Just one more question. I need to train it on a lot more characters. I see that U have only used small case ascii letters and 0-9. Will I need to retrain the whole model for that. Or I could just use the same model and somehow change the final layers. It would be great if you could help/guide me regarding that

**Comment by kurapan** [comment]
I see. A friend of mine trained a similar model for the Japanese language, which means a few thousand characters. He used it for text in simple font on simple background, and was able to achieve quite a good accuracy. 
I guess you could try using the pretrained model. It might speed up the training process because the model already has a notion of character strokes. Just replace the final layer, freeze all the other layers and train only the last layer until it starts producing somewhat sensible results, and then unfreeze the other layers and finetune the whole model.
